src/main/java/htsjdk/samtools/cram/structure/Slice.java

```diff
@@ -246,10 +257,14 @@ void setAttribute(final short tag, final Object value, final boolean isUnsignedA
         }
     }
 
-    public boolean isMapped() {
+    public boolean isMappedSingleRef() {
         return sequenceId > SAMRecord.NO_ALIGNMENT_REFERENCE_INDEX;
     }
 
+    public boolean isUnmapped() {
```

Comment:
This could use some comments on the possible interpretations (only unmapped records, or mixed, etc), especially since it uses a value from SAMRecord sequence ID state, but it doesn't mean the same thing as `SAMRecord` `unmapped` flag. Ultimately it seems like `Slice` and `Container` could each have a single state variable, possibly drawn from (separate ?) enums, that dictate which state they're in.